packages/htmlbars-runtime/lib/hooks.js

```diff
@@ -909,9 +914,9 @@ export function getCellOrValue(reference) {
   return reference;
 }
 
-export function component(morph, env, scope, tagName, params, attrs, template, visitor) {
+export function component(morph, env, scope, tagName, params, attrs, template, inverse, visitor) {
```

Comment:
htmlbars is chock full of functions that accept `template` and `inverse` as separate arguments. To do this right, we would change almost all of them to pass around a `templates` object instead. That looks like a big refactor. I'm going to limit my change to the `component` hook for now.